influence_benchmark/RL/run_KTO_iteration.py

```diff
@@ -87,23 +91,20 @@ def format_dataset(example):
         num_positives * kto_config.desirable_weight / (num_negatives * kto_config.undesirable_weight),
     )
 
-    if args.lora_path is not None:
-        model.load_adapter(args.lora_path, adapter_name="adapter_to_train")
-        model.load_adapter(args.lora_path, adapter_name="reference_adapter")
-    else:
-
-        model.add_adapter(peft_config, adapter_name="adapter_to_train")
-        model.add_adapter(peft_config, adapter_name="reference_adapter")
-
     trainer = KTOTrainer(
         model=model,
-        model_adapter_name="adapter_to_train",
         ref_adapter_name="reference_adapter",
         tokenizer=tokenizer,
         train_dataset=dataset,
         args=kto_config,
+        peft_config=peft_config,
```

Comment:
wait doesn't this mean it's training a new lora each time rather than starting from the previous checkpoint?

Reply:
we're only loading the previous lora as the reference, surely we also need to load it as the main model?